src/core/Darwin.zig

```diff
@@ -127,12 +175,19 @@ pub fn deinit(darwin: *Darwin) void {
     return;
 }
 
-pub fn update(_: *Darwin) !void {
-    return;
+pub fn update(self: *Darwin) !void {
+    if (self.window) |window| {
+        window.update();
+    }
```

Comment:
can reduce to just `if (darwin.window) |window| window.update();`

Reply:
Done.